app/flows/check_travel_during_coronavirus_flow/start.erb

```diff
@@ -44,4 +44,13 @@
   <p>If you’re travelling to England from within the Common Travel Area (England, Scotland, Wales and Northern Ireland, Ireland, the Channel Islands and the Isle of Man) and haven’t been anywhere else within the 10 days before you arrive, there are no entry requirements for coming into England.</p>
 
   <p>If you’ve been to a country or territory outside the Common Travel Area within the 10 days before you arrive in England, you must follow the rules for entering England from that country. You can use this tool to find out those rules.</p>
+
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Travelling to England from outside the Common Travel Area",
+    heading_level: 2,
+    font_size: "m",
+    margin_bottom: 4,
+  } %>
+
+  <p>If you live outside the Common Travel Area (England, Scotland, Wales and Northern Ireland, Ireland, the Channel Islands and the Isle of Man) and are travelling to England, check what you need to do to <a href="https://www.gov.uk/guidance/travel-to-england-from-another-country-during-coronavirus-covid-19">travel to England during coronavirus</a>.</p>
```

Comment:
Not a blocker: Should be a relative link.
```suggestion
  <p>If you live outside the Common Travel Area (England, Scotland, Wales and Northern Ireland, Ireland, the Channel Islands and the Isle of Man) and are travelling to England, check what you need to do to <a href="/guidance/travel-to-england-from-another-country-during-coronavirus-covid-19">travel to England during coronavirus</a>.</p>
```

Reply:
Updated link - thanks